### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by starver20**
Add filters functionality
Fix sidebar

### CODE REVIEW

**Review by Kevin-Solomon — APPROVED** [review]
Clean UI.
Nice active state on the chips.

File: src/App.css [hunk 1]
```diff
@@ -87,6 +87,7 @@
   overflow-y: auto;
   background-color: var(--background-color);
   color: var(--primary-color);
+  transition: all 1s;
 }
 
 /* .sidebar:hover,
```
Comment (AnkurChunekar) [inline src/App.css:93]: Remove comments

File: src/components/card/video-card/VideoCard.jsx [hunk 2]
```diff
@@ -11,23 +11,22 @@ const VideoCard = ({ video }) => {
...
-    videosDispatch,
-    video,
-    isInHistory
-  );
+  // const { callAsyncFunction: addToHistoryHandler } = useAsync(
+  //   addToHistory,
+  //   videosDispatch,
+  //   video,
+  //   isInHistory
+  // );
```
Comment (AnkurChunekar) [inline src/components/card/video-card/VideoCard.jsx:24]: These ones too

**Review by AnkurChunekar — APPROVED** [review]
Rest everything looks good to merge,
- Filter functionality works fine.
- great work @starver20

**Pull request merged**